solr/modules/hadoop-auth/src/test/org/apache/solr/client/solrj/impl/Krb5HttpClientUtils.java

```diff
@@ -36,10 +36,11 @@
   public static void setup(Http2SolrClient http2Client, String principalName) {
     HttpAuthenticationStore authenticationStore = new HttpAuthenticationStore();
     authenticationStore.addAuthentication(createSPNEGOAuthentication(principalName));
-    http2Client.getHttpClient().setAuthenticationStore(authenticationStore);
+    http2Client.setAuthenticationStore(authenticationStore);
     http2Client
         .getProtocolHandlers()
         .put(new WWWAuthenticationProtocolHandler(http2Client.getHttpClient()));
+    http2Client.setAuthenticationStore(authenticationStore);
```

Comment:
We call this twice?

Reply:
came in during rebase reconciliation I think, back to one